staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/resources.go

```diff
@@ -81,6 +83,36 @@ func NewNoxuCustomResourceDefinition(scope apiextensionsv1beta1.ResourceScope) *
 	}
 }
 
+// NewNoxuV1CustomResourceDefinition returns a WishIHadChosenNoxu CRD.
+func NewNoxuV1CustomResourceDefinition(scope apiextensionsv1.ResourceScope) *apiextensionsv1.CustomResourceDefinition {
+	return &apiextensionsv1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{Name: "noxus.mygroup.example.com"},
+		Spec: apiextensionsv1.CustomResourceDefinitionSpec{
+			Group: "mygroup.example.com",
+			Versions: []apiextensionsv1.CustomResourceDefinitionVersion{{
+				Name:    "v1beta1",
```

Comment:
the apisnoop detection gets confused by this when the e2e test calls a "beta" API.

After this merges, I think we'll want to switch all the custom resource e2e versions used to "v1", "v2", etc to ensure conformance tests use only "GA" versions.

Reply:
Interesting. I've added a line item in our sprint tracking so we remember to do this as a follow up.

Reply:
> the apisnoop detection gets confused by this when the e2e test calls a "beta" API.

ouch, do you know if we could fix this on that end ..?

Reply:
filtering out "*.example.com" APIs on that end seems reasonable to ask API snoop to do.

 The aggregated API server tests have the same issue (wardle.k8s.io/v1beta1 get flagged, etc), though I'd like to change those to something.example.com as well.

Reply:
cc @hh